.github/workflows/pr-stage-manage.yml

```
@@ -16,6 +17,7 @@ permissions:
   pull-requests: write
   issues: write
   contents: read
+  metadata: write
```

actionlint (1.7.12)

[error] unknown permission scope "metadata". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)